doc/lsst.synpipe/Synpipe_QuickStart.rst

```diff
@@ -0,0 +1,201 @@
+#########################
+Synpipe Quick Start Guide
+#########################
+
+Synpipe is a tool in the LSST software stack that is used to insert fake objects
+into data that is to be processed with the sofware stack. Though Synpipe is not
+the only way to insert fake objects into the stack, anyone can create a custom
+fake object tasks, it provides a quick and convenient method for the most common
+use cases. This document aims to make a user able to start using Synpipe in the
+shortest time possible while providing enough detail to understand the over all
+workflow.
+
+An important note is that this tool was initially created for the HSC survey,
+and although the LSST stack supports many different cameras, at this point Synpipe
+is currently limited to working with HSC data repositories.
+
+This guide will assume processing will be done though the pipe_drivers scripts.
+All the information is generally applicable, but configuration paths will change
+relative to what task is run in place of pipe_driver tasks.
+
+Generating A Catlaog
+====================
+
+For many of the fake object insertion tasks it is necessary to pre-generate a
+catalog of fake objects from which the insertion process can draw from. This is
+most important when fake objects are to be placed at specific locations in the
+sky such that they are inserted in the same place consistently in each input
+frame.
+
+Generating an input catalog for both stars and galaxies are done with the
+`makeSourceList.py` command, but differ on some of the inputs. For either type
```

Comment:
In reStructuredText, all these single backticks need to be double backticks to be recognized as code literals instead of Python API links.